### PR TITLE
Fixing github issues 9 and 10

### PR DESCRIPTION
**Pull request by attilapalfi92**
- faster hashcode function for LatLng
- using Math.round for better double to long conversion in LatLngConfig
- additionally made it compilable
- junit version update

### CODE REVIEW

**Comment by JavadocMD** [comment]
Just a note to say I greatly appreciate the pull request. (Despite its surprise closing.) I've begun the process of modernizing this repository in a number of ways, and I'll include the junit update and an improved hash function.

I found your fix rounding the double-to-long conversions resulted in a number of broken test cases, and I worry might alter expected functionality. However I have a theory that rounding in more limited contexts might be warranted, for example when doing radians-to-degrees math for comparison against the internal fixed-width representation.

Long story short, I am working on a version update to address the spirit of this pull request. Thank you for the inspirational nudge. :slightly_smiling_face:
